docs/storefront/graphql/multi-language-support.mdx

```diff
@@ -0,0 +1,114 @@
+# Multi-Language Support in GraphQL Storefront API
+
+The Storefront GraphQL API supports queries for multiple languages translations within a storefront channel. 
+
+You can query product information for locales in a storefront channel. For information on what product information we support in multiple langauges, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.
+
+## Prerequisites
+
+- **Create language transations:**
+
+  Set the product data for a locale using the GraphQL Admin API. For example queries, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.
+  <br />
+- **Choose a default locale:**
+
+  You can choose a default locale for the storefront channel, default to the global locale, or choose to override the default locale with the shopper browser language. Read more about how to [choose a default locale](https://support.bigcommerce.com/s/article/International-Enhancements-for-Multi-Storefront#localization) through the control panel. These locale settings affect which locale the API resolves.
+
+## Returned data
+
+The response includes the following:
+- **Enabled locales:** The list of the enabled locales for a channel. Includes the default locale and the shopper languages in the browser cookies.
+  <br />
+- **Resolved locale:** Indicates the locale that was selected to contain the returned data.
+
+  If you chose a default locale in the control panel, the resolved locale is the default locale.
+
+  If you chose to override the default locale with the shopper browser language, the API uses the browser's cookies to determine a shopper's language preference. To override shopper's preference, you can specify a preferred locale through the `Accept-Language` header or a GraphQL directive when you send the query. The directive takes precedence over the `Accept-Language` header<br />
+  <br />
+- **Product data for the resolved locale:** If there is no product data for the resolved locale, the response defaults to global product data.
```

Comment:
Not sure if this is accurate, if someone could confirm

Reply:
@jyuricle @andrii-lenets Can you confirm the statement above?

Reply:
@bc-andreadao yes, that's correct. The response will fallback to the global values if overrides are unavailable.

Reply:
Great, thanks